<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewGameDialog</class>
 <widget class="QDialog" name="NewGameDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>_(&quot;New game&quot;)</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>_(&quot;Game type&quot;)</string>
     </property>
     <layout class="QHBoxLayout">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>_(&quot;Mode:&quot;)</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>4</number>
        </property>
        <item>
         <widget class="QRadioButton" name="radioButtonDuplicate">
          <property name="text">
           <string>_(&quot;Duplicate&quot;)</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButtonFreeGame">
          <property name="text">
           <string>_(&quot;Free game&quot;)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButtonTraining">
          <property name="text">
           <string>_(&quot;Training&quot;)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButtonArbitration">
          <property name="text">
           <string>_(&quot;Arbitration (duplicate)&quot;)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButtonTopping">
          <property name="text">
           <string>_(&quot;Topping&quot;)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>16</width>
          <height>26</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>_(&quot;Variant:&quot;)</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QCheckBox" name="checkBoxJoker">
          <property name="text">
           <string>_(&quot;Joker game&quot;)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxExplosive">
          <property name="text">
           <string>_(&quot;Explosive game&quot;)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox7Among8">
          <property name="text">
           <string>_(&quot;7 among 8&quot;)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxMaster">
     <property name="title">
      <string>_(&quot;Master game&quot;)</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QCheckBox" name="checkBoxUseMaster">
        <property name="text">
         <string>_(&quot;Use a master game&quot;)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widgetMasterControls" native="true">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>_(&quot;Saved game to use:&quot;)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditMaster"/>
         </item>
         <item>
          <widget class="QPushButton" name="buttonBrowseMaster">
           <property name="text">
            <string>_(&quot;Browse...&quot;)</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxPlayers">
     <property name="title">
      <string>_(&quot;Players&quot;)</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QTableWidget" name="tablePlayers"/>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QPushButton" name="pushButtonAdd">
          <property name="text">
           <string>_(&quot;Add &amp;player&quot;)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonAddFav">
          <property name="text">
           <string>_(&quot;Add &amp;favorite players...&quot;)</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="buttonUp">
          <property name="toolTip">
           <string>_(&quot;Move selection upwards&quot;)</string>
          </property>
          <property name="icon">
           <iconset resource="../eliot.qrc">
            <normaloff>:/images/go-up.png</normaloff>:/images/go-up.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonDown">
          <property name="toolTip">
           <string>_(&quot;Move selection downwards&quot;)</string>
          </property>
          <property name="icon">
           <iconset resource="../eliot.qrc">
            <normaloff>:/images/go-down.png</normaloff>:/images/go-down.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>radioButtonDuplicate</tabstop>
  <tabstop>radioButtonFreeGame</tabstop>
  <tabstop>radioButtonTraining</tabstop>
  <tabstop>radioButtonArbitration</tabstop>
  <tabstop>radioButtonTopping</tabstop>
  <tabstop>checkBoxJoker</tabstop>
  <tabstop>checkBoxExplosive</tabstop>
  <tabstop>checkBox7Among8</tabstop>
  <tabstop>checkBoxUseMaster</tabstop>
  <tabstop>lineEditMaster</tabstop>
  <tabstop>buttonBrowseMaster</tabstop>
  <tabstop>pushButtonAdd</tabstop>
  <tabstop>buttonAddFav</tabstop>
  <tabstop>buttonUp</tabstop>
  <tabstop>buttonDown</tabstop>
  <tabstop>tablePlayers</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="../eliot.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewGameDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>461</y>
    </hint>
    <hint type="destinationlabel">
     <x>173</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewGameDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>461</y>
    </hint>
    <hint type="destinationlabel">
     <x>75</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
